### PR TITLE
[Fix] Remove duplicated Translations for `index.md` with unnecessary double quotes

### PR DESCRIPTION
**Pull request by ghost**
1) Remove duplicated Chinese translations for `index.md`, there's ONLY
one explaination.

2) Remove double quotes for `newsletter-postfix`. We don't need them and
the result seems the same.

image

### CODE REVIEW

**Comment by ghost** [comment]
@fhemberger ：
Thanks for marking mine :)

BTW：
1) Why your `@nodejs/nodejs-cn` is in BOLD but mine is a common style? I just copied from yours...
2) Are `newsletter-postfix`'s double quotation marks removed in `index.md` ?  I just had a test and it seems I didn't find anything different before they exist and after they are removed?
Or do you in fact want to use `&quot;` instead?

image
image

**Comment by fhemberger** [comment]
1. I used the GitHub autocompletion, typing `@nodejs…`, then it's turned into a notification (it's linked to the group)
2. Strings in YAML headers don't need quotation marks, they are stripped by the parser.

**Comment by Maledong** [comment]
OK. Thanks anyway and I'll remove them in English version. We don't need them any more.

**Comment by ghost** [comment]
PS：Since the main page `index.md` has been shortened to ONLY ONE explaination of Nodejs. Other translations should also be modified **by removing the last pharagraph**.

image

/cc：
@nodejs/nodejs-ar 
@nodejs/nodejs-tw 
@nodejs/nodejs-ja 
@nodejs/nodejs-uk 
@nodejs/nodejs-ko 
@nodejs/nodejs-de 
@nodejs/nodejs-it 
@nodejs/nodejs-fr 
@nodejs/nodejs-es

File: locale/zh-cn/index.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ labels:
   version-schedule-prompt-link-text: LTS 日程。
   newsletter: true
   newsletter-prefix: 订阅
-  newsletter-postfix: "，Node.js 官方的新闻周报。"
```
Comment (PeterDaveHello) [inline locale/zh-cn/index.md:21]: This change seems to be unrelated.

**Comment by vsemozhetbyt** [comment]
PR for the `locale/uk/index.md`: https://github.com/nodejs/nodejs.org/pull/1726

File: locale/zh-cn/index.md (hunk 1 above)
Comment (ghost) [reply locale/zh-cn/index.md:21]: Yes, I just mean that removing the last pharagraph, with the image pasted above for you to see. And there doesn't seem to need quotation marks, so I removed in this fix together.

Comment (PeterDaveHello) [reply locale/zh-cn/index.md:21]: Then maybe the PR title should be revised to cover that?

Comment (ghost) [reply locale/zh-cn/index.md:21]: OK. I'll update it. Thanks!
